reducers/gotPlayer/player.js

```diff
@@ -28,3 +31,19 @@ export default (state = initialState, action) => {
       return state;
   }
 };
+
+export const player = {
```

Comment:
Why is this necessary?  Can't you just point the component to the path in the state?